mllib/src/test/scala/org/apache/spark/ml/optim/WeightedLeastSquaresSuite.scala

```diff
@@ -43,6 +44,18 @@ class WeightedLeastSquaresSuite extends SparkFunSuite with MLlibTestSparkContext
       Instance(23.0, 3.0, Vectors.dense(2.0, 11.0)),
       Instance(29.0, 4.0, Vectors.dense(3.0, 13.0))
     ), 2)
+
+    /*
+       R code:
+       same as above except make the label constant
```

Comment:
IMO, comments referencing things like "above" should be avoided since the code can be rearranged. This comment would be confusing if the above data is declared in a different place or removed altogether. I don't think think pointing out that it is the same as the other data is necessary here.